docs/content/users/quickstart.md

```diff
@@ -70,6 +70,33 @@ Please note that you will need to change the PHP version to 7.4 to be able to wo
     ddev launch
     ```
 
+## CiviCRM (Standalone)
+
+[CiviCRM Standalone](https://civicrm.org/blog/ufundo/next-steps-civicrm-standalone) allows running [CiviCRM](https://civicrm.org/) without a CMS.
+
+```bash
+mkdir my-civicrm-site && cd my-civicrm-site
+ddev config --project-type=php --upload-dirs=public/media
+ddev start
+ddev exec "curl -LsS https://download.civicrm.org/latest/civicrm-STABLE-standalone.tar.gz -o /tmp/civicrm-standalone.tar.gz"
+ddev exec "tar --strip-components=1 -xzf /tmp/civicrm-standalone.tar.gz"
+ddev exec "curl -LsS https://download.civicrm.org/cv/cv.phar --create-dirs -o bin/cv && chmod +x bin/cv"
```

Comment:
Should we add more CLI tools from #6856? Ignore this change if you don't like it.
(I added `--no-scripts` because it didn't work otherwise.)

```suggestion
ddev composer require civicrm/cli-tools --no-scripts
```

Reply:
Thanks @stasadev, that's a great suggestion, having Composer take care of the placement, permissions, etc. is a more elegant solution.